backend/routes/policies.js

```diff
@@ -0,0 +1,61 @@
+const { Policies } = require('../models')
+const { authShop } = require('./_auth')
+
+/*
+Use the [2] lines below to enable logging to the Terminal
+const { getLogger } = require('../utils/logger')
+const log = getLogger('routes.policies')
+*/
+
+/*
+References:
+https://sequelize.org/master/manual/model-querying-basics.html
+https://expressjs.com/en/guide/routing.html
+*/
+
+module.exports = function (router) {
+  // Step one: Call the middleware function 'authShop' on any POST request to the endpoint '/shop/policies'
+  router.post('/shop/policies', authShop, async (req, res) => {
+    // Step two: Using the result of the previous function call, find or create an instance of the model 'Policies' in the DB
+    // Sequelize's 'findOrCreate' method creates a new entry in the DB if one didn't exist.
+    const policyEntry = await Policies.findOrCreate({
+      where: {
+        shopId: req.shop.id
+      },
+      defaults: {
+        allPolicies: [['', '', false]]
+      }
+    }).then((resultArray) => resultArray[0])
```

Comment:
Mixing await and .then makes the code hard to comprehend. Let's not mix await and .then